ui/src/containers/VolumePageContent.js

```diff
@@ -181,22 +181,22 @@ const VolumePageContent = (props) => {
     {
       selected: isOverviewPage,
       title: intl.translate('overview'),
-      onClick: () => history.push(`${match.url}/overview${query.toString() && `?${query.toString()}`}`),
+      onClick: () => history.replace(`${match.url}/overview${query.toString() && `?${query.toString()}`}`),
```

Comment:
We should keep using `history.push`

> The replace method is similar to push, but instead of adding a new location, it will replace the location at the current index. 

which we will lose the current location in history.

Same for all the tabs after.